src/url.js

```diff
@@ -0,0 +1,70 @@
+const FlowdockText = require('flowdock-text');
+
+function createTextNode(state, content, level) {
+  var token = new state.Token('text', '', 0);
+  token.content = content;
+  token.level = level;
+  return token;
+}
+
+function flowdockUrl(md, state) {
+  const tokens = state.tokens;
+  const arrayReplaceAt = md.utils.arrayReplaceAt;
+  
+  var nodes = [];
+  var token, children;
+
+  for (var i = 0; i < tokens.length; i++) {
```

Comment:
In ES6, you should be able to do something like 

    for(let token of tokens) {
        ...

Some thing with the loop below. Possible even better is an forEach with a function:

    tokens.forEach(function(token, index) {
        ... inside of previous for loop goes in here ...
    }